stablehlo/dialect/StablehloOps.td

```diff
@@ -2220,15 +2220,17 @@ def StableHLO_DotGeneralOp: StableHLO_ShapedInterfaceOp<"dot_general", [Pure]> {
     Example:
     ```mlir
     %result = stablehlo.dot_general %lhs, %rhs,
-      batching_dims = [0] x [0], contracting_dims = [2] x [1], precision = [DEFAULT, DEFAULT]
-      : (tensor<2x2x2xi32>, tensor<2x2x2xi32>) -> tensor<2x2x2xi32>
+      batching_dims = [0] x [0],
+      contracting_dims = [2] x [1],
+      precision = [DEFAULT, DEFAULT]
+      : (tensor<2x2x2xi64>, tensor<2x2x2xi64>) -> tensor<2x2x2xi64>
     ```
   }];
   let arguments = (ins
-    HLO_Tensor:$lhs,
-    HLO_Tensor:$rhs,
-    StableHLO_DotDimensionNumbers:$dot_dimension_numbers,
-    StableHLO_PrecisionConfigAttr:$precision_config
+    HLO_Tensor:$lhs /*dot_general_i1*/,
+    HLO_Tensor:$rhs /*dot_general_i2*/,
+    StableHLO_DotDimensionNumbers:$dot_dimension_numbers /*dot_general_i3, dot_general_i4, dot_general_i5, dot_general_i6*/,
```

Comment:
The PR description says that i3a, i4a, i5a and i6a are not covered by the ODS, but I think they are.

Reply:
Ack.

Reply:
These test cases still show up under "If we drop the "Covered by ODS" pieces, this will leave us with the following test cases".

Reply:
They should be gone and may not be updated on your side. Try refreshing the page?